src/noit_check_resolver.c

```diff
@@ -532,6 +532,10 @@ int noit_check_resolver_loop(eventer_t e, int mask, void *c,
   return 0;
 }
 
+static inline void start_noit_check_resolver_loop() {
+  eventer_add_in_s_us(noit_check_resolver_loop, NULL, 1, 0);
```

Comment:
Why wait 1 second? Why not 0?

Reply:
Was copied from before. Really no reason for it not to be zero. I've changed it to 0.